portal/src/main/scala/cromwell/pipeline/CromwellPipelineApp.scala

```diff
@@ -21,6 +23,18 @@ object CromwellPipelineApp extends App {
   val log = LoggerFactory.getLogger(CromwellPipelineApp.getClass)
   val components = new ApplicationComponents()
 
+  log.info(
+    Json.prettyPrint(
+      Json.obj(
+        "auth" -> Json.toJson(ApplicationConfig.load().authConfig),
+        "webservice" -> Json.toJson(ApplicationConfig.load().webServiceConfig),
+        "mongo" -> Json.toJson(ApplicationConfig.load().mongoConfig),
+        "postgres" -> Json.toJson(ApplicationConfig.load().postgresConfig),
+        "gitlab" -> Json.toJson(ApplicationConfig.load().gitLabConfig)
```

Comment:
1) No need to read whole Application config every time from scratch since all of them are already available in `component` variable.
2) Putting so much logic about config printing inside initialization object is wrong. For example, you can create something like `ConfigRenderer` object in utils, that will have only one method
`def render(configs: ConfigComponent*): String`
And then rewrite line 26 as
```
import components.applicationConfig._
log.info(ConfigRenderer.render(authConfig, webServiceConfig, mongoConfig, postgresConfig, gitLabConfig))
```